native/src/routes/SearchModal.tsx

```diff
@@ -112,7 +151,7 @@ const SearchModal = ({
       <SearchHeader theme={theme} query={query} closeSearchBar={onClose} onSearchChanged={setQuery} t={t} />
       <KeyboardAvoidingView behavior={Platform.OS === 'ios' ? 'padding' : undefined} style={{ flex: 1 }}>
         <List
-          items={searchResults}
+          items={query.length > 0 ? searchResults : allPossibleResults}
```

Comment:
Is this necessary? Doesn't minisearch return all items if the query is empty?

Reply:
No, it unfortunately returns an empty array then.